src/routes/blog/post/role-based-access-control-with-appwrite/+page.markdoc

```diff
@@ -73,7 +73,7 @@ In summary, roles map users to the permissions they need, thus streamlining the
 
 **Mandatory Access Control (MAC)** uses a centralized authority to decide access, often based on classifications. It's strict and primarily used in environments requiring high security, such as government systems.
 
-**Attribute-Based Access Control (ABAC)** relies on dynamic policies using attributes, such as user roles and environmental conditions. This model excels in environments needing fine-grained controls.
+**Column-Based Access Control (CBAC)** relies on dynamic policies using columns, such as user roles and environmental conditions. This model excels in environments needing fine-grained controls.
```

Comment:
ABAC is a standard access control model; the term has nothing to do with Appwrite Databases